UI/volume-control.cpp

```diff
@@ -832,7 +853,10 @@ void VolumeMeter::paintEvent(QPaintEvent *event)
 			5, channelNr * 4, width - 5, 3,
 			displayMagnitude[channelNr], displayPeak[channelNr],
 			displayPeakHold[channelNr]);
-
+		
```

Comment:
Please don't uncomment this. It is important for the little box in the front of the meter to be drawn for debugging audio problems.

Reply:
No se como funciona i nose ni que e tocao

Reply:
Que ago para arreglarlo